scripts/lib.sh

```diff
@@ -22,7 +22,11 @@ function open_channel() {
 }
 
 function await_fedimint_block_sync() {
-  fixtures await-fedimint-block-sync
+  BLOCKS="$($FM_BTC_CLIENT getblockchaininfo | jq -e -r '.blocks')"
+  FINALITY_DELAY=$(get_finality_delay)
+  AWAIT="$((BLOCKS - FINALITY_DELAY))"
+  echo "await_fedimint_block_sync $AWAIT"
+  $FM_MINT_CLIENT wait-block-height "$AWAIT"
```

Comment:
This wasn't working. @Maan2003 should be able to kill this for good pretty soon!

Reply:
It was working just redundant to have this in Rust twice.  Think we should just convert the `setup-tmuxinator` to be part of `fedimint-bin-tests` then we can eliminate most of the remaining bash scripts.